### DISCUSSION TITLE
1.6 版本更新相关问题

### DISCUSSION THREAD

**Discussion by weineo** [libnyanpasu/clash-nyanpasu#1468] [General]
目前的1.6版本个人遇见的小问题：
### 一、更新后旧版本的所有配置文件不可用。
虽然配置文件夹内可见旧有配置文件，但软件界面不可见。
### 二、新版本的各种对订阅进行处理的配置文件如何建立不清晰，文档似乎也未建立。
原有js脚本等“进阶”配置，暂时未找到在新版本中如何使用。软件内如何新建merge、script文件，未找到相关功能模块，也未在官方文档里见到说明

干脆直接回退旧版本了，等待新版本的文档建立完善，功能介绍清晰，搞清楚怎么从旧版本完善的切换至新版本后，再行更新。
个人情况，N个通过链接导入的remote类型文件。加上两个分别替代基础配置、代理规则的merge文件，以及一个处理并重新组织代理组的script文件。

强烈建议和我一样的跟着众多大佬搞了很多进阶个性化配置的小白暂时不要更新，目前切换非常不顺滑。单纯用订阅链接导入而未进行任何个性化设置的真~纯小白倒是可以更新，UI确实进步了很多

**Comment by HozumiMoa** [comment]
你把所有个性化的设置全部维护到一个 yaml 文件里，就没那么多麻烦事了

**Comment by CPCer** [comment]
后悔现在才看到，因为bug更新后，配置全没了。关键以前各个配置是什么意思，现在全部记得了，还得冲头开始搞

**Reply by greenhat616** [reply]
请不要删除 Profiles，配置可以恢复。切记不要执行重新添加 Profile 的步骤，会覆盖配置文件。
该 Bug 疑似迁移脚本遗漏了自定义脚本类型字段，之后计划通过新增一个迁移脚本修复此问题。

以下为手动迁移步骤：
找到配置目录下的 `profiles.yaml`，
搜索替换`type: script` 为 `type: !script javascript`。
然后重启应用即可。
